cli/flox-rust-sdk/src/models/environment/remote_environment.rs

```diff
@@ -95,17 +95,17 @@ impl RemoteEnvironment {
 #[async_trait]
```

Comment:
Do we still need this?

Reply:
Yeah we have the catalog method 
Though we ain't using it
Should just remove it tbh

Reply:
I would just remove it if we aren't using it

Reply:
Catalogs are getting an overhaul anyway, we don't know what the implementation is going to look like, and we can always do a `spawn_blocking` if we have one async block to execute.

Reply:
yeah i didnt keep it in for any reason, rahter for not having an excuse to delete it :D
this might be it now...

Reply:
removed